crates/typst/src/foundations/array.rs

```diff
@@ -724,6 +724,41 @@ impl Array {
         Array(vec)
     }
 
+    /// Splits an array into non-overlapping chunks, starting at the beginning,
+    /// ending with a single remainder chunk.
+    ///
+    /// All chunks but the last have `chunk_size` elements.
+    /// If `exact` is set to `{true}`, the remainder is dropped if it
+    /// contains less than `chunk_size` elements.
+    ///
+    /// ```example
+    /// #let array = (1, 2, 3, 4, 5, 6, 7, 8)
+    /// #array.chunks(3)
+    /// #array.chunks(3, exact: true)
+    /// ```
+    #[func]
+    pub fn chunks(
+        self,
+        /// How many elements each chunk may at most contain.
+        chunk_size: i64,
```

Comment:
can use `NonZeroUsize` here and then the function doesn't need to return a result anymore

Reply:
the second part wasn't addressed, the function still returns a result, but now unnecessarily